ansible_collections/arista/avd/docs/pyavd.md

```diff
@@ -31,7 +28,7 @@ Feedback is very welcome. Please use [GitHub discussions](https://github.com/ari
 ## Known limitations
 
 - No support for inline Jinja2 or custom Jinja2 templates.
-- Input data and "structured_configs" will be in-place updated by various pyavd functions.
+- Input data and "structured_configs" will be in-place updated by various PyAVD functions.
   Make sure to deep copy the data first if modifications are not allowed.
```

Comment:
I would make this a bit more "visible" maybe a note or .. I don't know to draw attention